gtest-parallel

```diff
@@ -266,6 +266,15 @@ parser.add_option('--format', type='string', default='filter',
                   help='output format (raw,filter)')
 parser.add_option('--print_test_times', action='store_true', default=False,
                   help='When done, list the run time of each test')
+parser.add_option('--shard_count', type='int',
+                  default=int(os.environ.pop('GTEST_TOTAL_SHARDS', 1)),
+                  help=('Total number of shards (for sharding test execution '
+                        'between multiple machines). Default: %default'))
+parser.add_option('--shard_index', type='int',
+                  default=int(os.environ.pop('GTEST_SHARD_INDEX', 0)),
+                  help=('Zero-indexed number identifying this shard (for '
+                        'sharding test execution between multiple machines). '
+                        'Default: %default'))
```

Comment:
Agreed, calling this with environment variables set on Windows/Linux  seems to be not impossible. This is a huge foot-shooter, can you do os.environ.get() in the chromium buildbot script outside that calls the gtest-parallel binary and simply specify those as command-line args?